<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="leftGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Photo Staging Area</string>
    </property>
    <widget class="QPushButton" name="stageButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stage</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="styleTwoCheckBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cartoonify</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="styleOneCheckBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Borders</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>390</width>
       <height>330</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rightGroupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Photo Minting Area</string>
    </property>
    <widget class="QLabel" name="rightLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>390</width>
       <height>330</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="mintButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>MINT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInputBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>NFT name...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionInputBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>NFT Description...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionDocs">
   <property name="text">
    <string>Help Document</string>
   </property>
  </action>
 </widget>
 <!---
 <resources>
    <include location="test.qrc"/>
 </resources>
 -->
 <connections/>
</ui>
